.changeset/dry-teachers-doubt.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
@anshmourya could you please add a changeset with description?

run `changeset` select the navbar package and add the description

Fix #NUMBER_OF_ISSUE, explanation..

Push the new changeset

Thank you!